1.8/overview/telemetry.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
Yes, this is a separate issue: https://mesosphere.atlassian.net/browse/DOCS-1063